### PR TITLE
Pt pt translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
## Purpose

[TRANSLATION] Portuguese Translation 

 [WIN32][NETID] Improvements to Portuguese translation
[DLL][SYSDM] Update Portuguese translation and some object adjustments
[WIN32][SHIMGVW] New Portuguese translation

### CODE REVIEW

File: dll/cpl/sysdm/lang/pt-PT.rc [hunk 1]
```diff
@@ -72,9 +72,9 @@ CAPTION "Definições de sistema"
 FONT 8, "MS Shell Dlg", 0, 0, 0x1
 BEGIN
     GROUPBOX "Informações de versão", IDC_STATIC, 6, 3, 210, 73
-    LTEXT "O ReactOS é construído como servidor de sistemas operativos  e responde como tal. Marque esta opção para seleccionar apenas aplicativos.", IDC_STATIC, 15, 15, 183, 41
+    LTEXT "O ReactOS é construído como servidor de sistemas operativos  e responde como tal. Marque esta opção para aplicar isso apenas a aplicativos.", IDC_STATIC, 15, 15, 183, 41
```
Comment (HBelusca) [inline dll/cpl/sysdm/lang/pt-PT.rc]: After `operativos` and before `e responde` (first sentence), there is an extra space too much.

File: dll/cpl/sysdm/lang/pt-PT.rc [hunk 2]
```diff
@@ -175,177 +175,177 @@ CAPTION "Change Profile Type"
...
-CAPTION "Virtual Memory"
+CAPTION "Memória virtual"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-    LTEXT "Drive  [Volume Label]", IDC_STATIC, 12, 5, 96, 9
-    LTEXT "Paging File Size (MB)", IDC_STATIC, 115, 5, 96, 9
-    LISTBOX IDC_PAGEFILELIST, 10, 16, 204, 52, LBS_NOTIFY | LBS_USETABSTOPS
-    GROUPBOX "Paging file size for selected drive", IDC_DRIVEGROUP, 10, 70, 204, 104
-    LTEXT "Drive:", IDC_STATIC, 20, 80, 20, 8
+    LTEXT "Unidade[Nome de volume]", IDC_STATIC, 12, 5, 96, 9
```
Comment (HBelusca) [inline dll/cpl/sysdm/lang/pt-PT.rc]: missing space after "Unidade" ?

File: dll/win32/shimgvw/lang/pt-PT.rc [hunk 3]
```diff
@@ -0,0 +1,33 @@
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
+/* Portuguese Language resource file
```
Comment (Fraizeraust) [inline dll/win32/shimgvw/lang/pt-PT.rc]: Usually the comment header goes at the top -- just for your information.

Comment (zecarlos1957) [reply dll/win32/shimgvw/lang/pt-PT.rc]: You're absolutely right, it was unintentional. Thanks

**Pull request merged**